migrations/bootstrap.sql

```diff
@@ -19,6 +19,7 @@ CREATE TABLE `applicants` (
   `heard_from` TEXT,
   `profile_hash` char(40) DEFAULT NULL COMMENT 'For profile image lookup',
   `website` VARCHAR(255) DEFAULT NULL,
+  `preferred_gender_pronoun` VARCHAR(255) DEFAULT NULL,
```

Comment:
I think you'll want to remove this since the changeset covers it. I think. It's been a while :S